src/components/form/templates/object-field.ts

```diff
@@ -35,13 +35,26 @@ function renderCollapsibleField(props: LimeObjectFieldTemplateProps) {
         'limel-collapsible-section',
         {
             header: props.title,
+            id: getSchemaObjectPropertyPath(
+                props.formContext.schema,
+                props.idSchema
+            ),
             'is-open': defaultOpen,
         },
         renderDescription(props.description),
         renderProperties(props.properties, props.schema)
     );
 }
 
+function getSchemaObjectPropertyPath(schema: any, subSchema: LimeJSONSchema) {
+    const rootStart = 2;
+    const re = /\//g;
+    const rootPath = (schema.$ref as string).replace(re, '.').slice(rootStart);
+    const subSchemaPath = subSchema.$id?.replace('_', '.properties.');
```

Comment:
the ObjectFieldTemplate will only be applied to objects and hence the schema hierarchy (id) string should replace underscores with "properties". There is a corner case here where the schema hierarchy includes an array (or multiple). In that case the property path will be incorrect. Not sure how RJSF will build the schema hierarchy in that case..?

Example $ref: "#/definitions/CompanySchema"
Example id: "root_section1"
Example output: "definitions.CompanySchema.properties.section1"

Reply:
I realise now that the example I post there are the positive case. I just wanted to give you guys some context of what is actually sent in as props from RJSF but its related to my listed corner case.

>I think I understand the normal case, at least if the examples are for the normal case… but I'm not sure I understand this corner case you mention. Can you explain and give some examples for that?

Sure, let's say you have a nested schema that looks something like this (simplified):
CompanySchema = {
  someSection = [
    somethingCollapsible,
    somethingCollapsible,
    somethingCollapsible]
}

How will the $id schema hierarchy look like for element 1, 2 and 3 of someSection? I know that the wanted property path for the third `somethingCollapsible` should be `CompanySchema.someSection[2].somethingCollapsible`.

Maybe this is a bad example as they might be funnelled into the `ArrayFieldTemplate`? 🤔
But hopefully you get my point of that schema would be even more nested? Or perhaps my concern here are none existing 😄

We talked briefly about this @jgroth maybe you can weigh in?

Reply:
It's not really nested objects I'm worried about, that *should* be fine. It's more if there's an array in the middle. Then we might miss to substitute whatever nifty representation RJSF has for arrays into square brackets accessor.

I think I'm bad at explaining this. 😔
Maybe a call tomorrow? 📞